kombu/transport/redis.py

```diff
@@ -768,14 +770,21 @@ def _size(self, queue):
         with self.conn_or_acquire() as client:
             with client.pipeline() as pipe:
                 for pri in self.priority_steps:
-                    pipe = pipe.llen(self._q_for_pri(queue, pri))
+                    queue_name = self._queue_for_priority(queue, pri)
+                    pipe = pipe.llen(queue_name)
                 sizes = pipe.execute()
-                return sum(size for size in sizes
+                size = sum(size for size in sizes
                            if isinstance(size, numbers.Integral))
+                return size
 
-    def _q_for_pri(self, queue, pri):
+    def _queue_for_priority(self, queue, pri):
         pri = self.priority(pri)
-        return '%s%s%s' % ((queue, self.sep, pri) if pri else (queue, '', ''))
+        if pri:
+            queue_args = (queue, self.sep, pri)
+        else:
```

Comment:
I would suggest changing the var name `pri` from line 781 to other name, that's because you're defining a new var with the same name from your parameter

Reply:
Good idea, I'll update it to be `priority`.